samcli/lib/providers/provider.py

```diff
@@ -88,14 +88,24 @@ class Function(NamedTuple):
     @property
     def full_path(self) -> str:
         """
-        Return the path-like identifier of this Function. If it is in root stack, full_path = name.
+        Return the path-like identifier of this Function. If it is in root stack, full_path = function_id.
         This path is guaranteed to be unique in a multi-stack situation.
         Example:
             "HelloWorldFunction"
             "ChildStackA/GrandChildStackB/AFunctionInNestedStack"
         """
         return get_full_path(self.stack_path, self.function_id)
 
+    @property
+    def full_name(self) -> str:
+        """
+        Return the path-like name of this Function. If it is in root stack, full_name = name.
+        Example:
+            "HelloWorldFunction"
+            "ChildStackA/GrandChildStackB/AFunctionInNestedStack"
+        """
+        return get_full_name(self.stack_path, self.name)
```

Comment:
I think this function will mix between the CDK ID, and logical ID. It should return the same value as full_path in CFN use cases, but in CDK use case it will return the <stack CDK ID>/<Function Logical ID>. I am not sure how this mix will help the customer to identify a lambda function.

Reply:
You are correct.
I will fix to using only CDK ID.